src/borg_import/helpers/testsuite/test_timestamps.py

```diff
@@ -18,7 +18,14 @@ def test_datetime_from_mtime(tmpdir):
 
 
 def test_datetime_from_string():
-    assert datetime_from_string('1999-12-31T23:59:59') == datetime(1999, 12, 31, 23, 59, 59)
-    assert datetime_from_string('Mon Oct 31 23:35:50 UTC 2016') == datetime(2016, 10, 31, 23, 35, 50)
+    # Start with local timezone, convert to UTC, and then convert back to local.
+    assert datetime_from_string('1999-12-31T23:59:59') == datetime(1999, 12, 31, 23, 59, 59).astimezone()
+    # FIXME: strptime discards timezone info, and creates a naive time.
+    # UTC is handled specially, both in strptime and datetime_from_string;
+    # local conversions using this format may or may not work.
+    assert datetime_from_string('Mon Oct 31 23:35:50 UTC 2016') == datetime(2016, 10, 31, 23, 35, 50,
+                                                                            tzinfo=timezone.utc)
+    # rsync-time-backup format.
+    assert datetime_from_string('2022-12-21-063019') == datetime(2022, 12, 21, 6, 30, 19).astimezone()
```

Comment:
lhs: starts from string in local tz, returns a tz-aware dt object in utc
rhs: starts with a naive datime object and converts that to a tz-aware dt object in **local timezone** on the rhs - why not in utc timezone (see lhs)?

Reply:
Suggestion: change rhs to `datetime(2022, 12, 21, 6, 30, 19).astimezone(tz=timezone.utc)`

Reply:
Suggestion applied. I added `assert dfs.tzinfo == dt_trg.tzinfo == timezone.utc` tests to ensure that we're always in the UTC timezone. Of course, two times in different timezones can be equal, but we want `borg` to ingest UTC-only.